src/main/java/com/service/AuthorisationService.java

```diff
@@ -0,0 +1,33 @@
+package com.service;
+
+import com.dao.UserDAO;
+import lombok.Data;
+
+import javax.sql.DataSource;
+
+@Data
```

Comment:
See previous comment about @Data annotation

Reply:
removed

Reply:
We need access to DAO within Service class.
Otherwise we need to set source of connection inside class.